vendor/k8s.io/kubernetes/staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go

```diff
@@ -690,19 +690,38 @@ func (l *terminationLoggingListener) Accept() (net.Conn, error) {
 		return nil, err
 	}
 
+	// ignore connections to local IP. Those clients better know what they are doing.
+	local := false
+	host, _, err := net.SplitHostPort(c.LocalAddr().String())
+	if err != nil {
+		// ignore error and keep going
+	} else if ip := net.ParseIP(host); ip == nil {
+		return c, nil
```

Comment:
what does it mean ? the host address is broken ?

Reply:
more like `// THIS SHOLD NEVER HAPPEN`.